### DISCUSSION TITLE
i18n suggestion

### DISCUSSION THREAD

**Discussion by jiimmyy1** [withastro/roadmap#787] [Proposal]
### Body

Due to the new release in Astro 3.5 of the i18n function, I wanna suggest something that I thing it's veeery important.

Now, if you want to make a change you have to do it in **ALL** your websites as much as locales you have.
My suggestion is to make just a page, and in a locales file, probably on JSON, add there the translations.

It would be great to see that on Astro.

**Comment by zanhk** [comment]
I agree as well, right now there is much duplicate code and if you have multiple languages could be a problem to mantain, right now I m using https://github.com/jlarmstrongiv/astro-i18n-aut but it would be nice if in astro i18n would work out of the box.
